tests/parser/ScheduleTests.cpp

```diff
@@ -391,6 +391,16 @@ BOOST_AUTO_TEST_CASE(CreateScheduleDeckWellsOrdered) {
     BOOST_CHECK_EQUAL( "CG", group_names[1]);
     BOOST_CHECK_EQUAL( "BG", group_names[2]);
     BOOST_CHECK_EQUAL( "AG", group_names[3]);
+
+    auto restart_groups = schedule.restart_groups(0);
+    BOOST_CHECK_EQUAL(restart_groups.size(), 4);
```

Comment:
This should probably be `BOOST_REQUIRE_EQUAL` since the subsequent code depends on `restart_groups` being non-empty.

Reply:
done